src/Check/HttpService.php

```diff
@@ -13,42 +13,27 @@
 use function fwrite;
 use function preg_match;
 use function sprintf;
-use function strpos;
+use function str_contains;
 
 /**
  * Attempt connection to given HTTP host and (optionally) check status code and page content.
  */
 class HttpService extends AbstractCheck
 {
-    /** @var string */
-    protected $host;
-
-    /** @var int */
-    protected $port;
-
-    /** @var string */
-    protected $path;
-
-    /** @var int */
-    protected $statusCode;
-
-    /** @var int */
-    protected $content;
-
     /**
      * @param string $host       Host name or IP address to check.
      * @param int    $port       Port to connect to (defaults to 80)
      * @param string $path       The path to retrieve (defaults to /)
      * @param int    $statusCode (optional) Expected status code
-     * @param null   $content    (optional) Expected substring to match against the page content.
+     * @param string $content    (optional) Expected substring to match against the page content.
```

Comment:
`string|null`?

Reply:
Updated to use `string|null` https://github.com/laminas/laminas-diagnostics/pull/68/commits/b075a902d53376d034b6b056eb431c1426754f07